patternlab/styleguide/source/assets/js/modules/utilNav.js

```diff
@@ -1,77 +1,79 @@
-export default (function (window, document, $, undefined) {

```

Comment:
@clair0917 I think this `export` wrapper is necessary to allow jQuery selectors to function in Drupal. Otherwise I'm getting an `Uncaught TypeError: $ is not a function` error.